addons/stock/models/res_config_settings.py

```diff
@@ -53,6 +53,18 @@ class ResConfigSettings(models.TransientModel):
         "Separator", config_parameter='stock.barcode_separator',
         help="Character(s) used to separate data contained within an aggregate barcode (i.e. a barcode containing multiple barcode encodings)")
     module_stock_fleet = fields.Boolean("Dispatch Management System")
+    replenish_on_order = fields.Boolean("Replenish on Order (MTO)", config_parameter='replenish_on_order_settings', compute='_compute_replenish_on_order', inverse='_inverse_replenish_on_order')
+
+    @api.depends('replenish_on_order')
+    def _compute_replenish_on_order(self):
+        route = self.env.ref('stock.route_warehouse0_mto')
+        if route:
+            self.replenish_on_order = route.product_categ_selectable = route.active
+
+    def _inverse_replenish_on_order(self):
+        route = self.env.ref('stock.route_warehouse0_mto')
+        if route:
+            route.active = route.product_categ_selectable = self.replenish_on_order
```

Comment:
```suggestion
            route.active = self.replenish_on_order
```
And add it at the same place where product_selectable is set

Reply:
I added it to `stock_data` instead because Product_selectable is True by default its not explicitly set somewhere else